scss/_variables.scss

```diff
@@ -1066,31 +1066,33 @@ $breadcrumb-border-radius:          $border-radius !default;
 
 // Carousel
 
-$carousel-control-color:             $white !default;
+$carousel-color:                     $white !default;
+
+$carousel-control-color:             inherit !default;
 $carousel-control-width:             15% !default;
 $carousel-control-opacity:           .5 !default;
 $carousel-control-hover-opacity:     .9 !default;
 $carousel-control-transition:        opacity .15s ease !default;
 
-$carousel-indicator-width:           30px !default;
-$carousel-indicator-height:          3px !default;
-$carousel-indicator-hit-area-height: 10px !default;
-$carousel-indicator-spacer:          3px !default;
-$carousel-indicator-active-bg:       $white !default;
+$carousel-indicator-width:           2rem !default;
+$carousel-indicator-height:          .25rem !default;
+$carousel-indicator-hit-area-height: .25rem !default;
+$carousel-indicator-spacer:          .25rem !default;
+$carousel-indicator-active-bg:       inherit !default;
 $carousel-indicator-transition:      opacity .6s ease !default;
 
 $carousel-caption-width:             70% !default;
-$carousel-caption-color:             $white !default;
+$carousel-caption-color:             inherit !default;
 
-$carousel-control-icon-width:        20px !default;
+$carousel-control-icon-width:        3rem !default;
+$carousel-control-icon-color:        currentColor !default;
```

Comment:
This won't work. Background images don't know what the `currentColor` is. Alternatives:
- Use an `<svg>` in the DOM
- Work with a rotated pseudo element which is rotated